### DISCUSSION TITLE
FSR support?

### DISCUSSION THREAD

**Discussion by thekillerwave** [skyline-emu/skyline#1470] [Ideas]
The RPCS3 emulator uses AMD's Fidelity Fx Super Resolution for better performance, would you, the developpers consider implementing it on Skyline?

**Comment by yhak22** [comment]
I think this is what docked and handheld doing.

**Reply by thekillerwave** [reply]
can you precise? i think that when the switch is docked, it has just more electricity so more power, this is why you get 1080p native docked and 720p handheld. FSR on the other point is an upscaling technic wich gives more performance without more watts needed

**Reply by yhak22** [reply]
oh , i see what you mean.
so u need upscaling or image enhanceing ?
maybe , but not at this moment cause the app is still in early stage .
so wait for it to be stable and powerfull enough.
